packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1006,6 +1018,7 @@ export class MgtPeoplePicker extends MgtTemplatedTaskComponent {
             people = groups;
           }
           this.defaultPeople = people;
+          people = this.disableSuggestions ? [] : people;
```

Comment:
this will allow the component to make calls to load the suggestions when there is no input provided.

When `disable-suggestions` is true and `input.length < 1` then we shouldn't make any call to graph apis.